docs/services/dicomweb-server.md

```diff
@@ -0,0 +1,24 @@
+# DICOMweb server
```

Comment:
Is this a service? I would had thought it should be documented as an endpoint as we only have this during testing

Reply:
Ah yeah good point, I might just move this to `pixl_core/README`